addons/account/models/account_journal.py

```diff
@@ -542,6 +498,14 @@ def copy(self, default=None):
         return super(AccountJournal, self).copy(default)
 
     def write(self, vals):
+        # for journals, force a readable name instead of a sanitized name e.g. non ascii in journal names
+        if vals.get('alias_name') and 'type' not in vals:
+            self.ensure_one()  # cannot set same name on several journals, unicity constraint
+            if (not self.env['mail.alias']._is_encodable(vals['alias_name']) or
+                not self.env['mail.alias']._sanitize_alias_name(vals['alias_name'])):
+                vals['alias_name'] = self._alias_prepare_alias_name_from_vals(
```

Comment:
"_alias_prepare_alias_name" ?

"from_vals" is a bit strange, I think we understand you will use provided arguments ^^

EDIT: well, looks like there is both, would honestly only keep the one that receives value and hardcode the other usage, as not re-used anyway.
(Not sure if clear).